tests/test_international_billing_rates.py

```diff
@@ -30,3 +31,11 @@ def test_international_billing_rates_are_in_correct_format(country_prefix, value
 
 def test_country_codes():
     assert len(COUNTRY_PREFIXES) == 214
+
+
+@pytest.mark.parametrize("number, expected",
+                         [('48123654789', False),
+                          ('140312345687', True),
```

Comment:
What is this number? I can't see a country code starting '140' in our yaml file or on https://en.wikipedia.org/wiki/List_of_country_calling_codes#Zone_1:_North_American_Numbering_Plan and the same for just '14'

Reply:
Added comments and some numbers that are not normalised